### PR TITLE
Object mapper support comments, trailing commas and single quote

### PR DESCRIPTION
**Pull request by johnksv**
Solves https://github.com/InseeFrLab/onyxia-api/issues/334

Implement features as described in fasterxml https://github.com/FasterXML/jackson-core/wiki/JsonReadFeatures

### CODE REVIEW

File: onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java [hunk 1]
```diff
@@ -14,26 +20,29 @@ public class CustomObjectMapper {
...
+        JsonMapper.Builder mapper = JsonMapper.builder();
         commonConfiguration(mapper);
-        return mapper;
+        return mapper.build();
     }
 
     @Bean
     @Qualifier("helm")
     public ObjectMapper objectMapperHelm() {
-        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```
Comment (johnksv) [inline onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java:25]: One aspect I did not consider is how the helm object mapper is effected by these changes. Any ideas? I mean, in yaml, most of the new features are supported anyway

Comment (olevitt) [reply onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java:25]: Considering the yaml mapper is only used when reading `Chart.yaml` and writing `values.yaml` I'm confident this change will be harmless. Nonetheless we could exclude this configuration from the YAML mapper as it's irrelevant to it and only apply it to the JSON mapper (move the new code out of `commonConfiguration` method).  
What do you think ?

**Review by olevitt — CHANGES_REQUESTED** [review]
See my comment about the YAML mapper, what do you think ?

File: onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java (hunk 1 above)
Comment (johnksv) [reply onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java:25]: I agree. I will fix it.

Comment (johnksv) [reply onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java:25]: Solved in https://github.com/InseeFrLab/onyxia-api/pull/344/commits/3e647e49e59b697bd8be98a4d15b573b729b7f83

**Review by olevitt — APPROVED** [review]
LGTM, thanks :+1:

**Pull request merged**